src/agent/onefuzz-supervisor/src/scheduler.rs

```diff
@@ -35,6 +35,28 @@ impl fmt::Display for Scheduler {
     }
 }
 
+pub enum StateValue {
```

Comment:
I think this is redundant. We can just nose [`NodeState`](https://github.com/microsoft/onefuzz/blob/main/src/agent/onefuzz-supervisor/src/coordinator.rs#L52).